bigtable/snippets/src/test/java/com/example/bigtable/FiltersTest.java

```diff
@@ -0,0 +1,738 @@
+                            TIMESTAMP,
+                            1)
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_wifi".getBytes()),
+                            TIMESTAMP,
+                            1)
+                        .setCell(COLUMN_FAMILY_NAME_STATS, "os_build", TIMESTAMP, "PQ2A.190405.004")
+                        .setCell(COLUMN_FAMILY_NAME_DATA, "data_plan_05gb", TIMESTAMP, "true"))
+                .add(
+                    "phone#4c410523#20190505",
+                    Mutation.create()
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_cell".getBytes()),
+                            TIMESTAMP,
+                            0)
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_wifi".getBytes()),
+                            TIMESTAMP,
+                            1)
+                        .setCell(COLUMN_FAMILY_NAME_STATS, "os_build", TIMESTAMP, "PQ2A.190406.000")
+                        .setCell(COLUMN_FAMILY_NAME_DATA, "data_plan_05gb", TIMESTAMP, "true"))
+                .add(
+                    "phone#5c10102#20190501",
+                    Mutation.create()
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_cell".getBytes()),
+                            TIMESTAMP,
+                            1)
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_wifi".getBytes()),
+                            TIMESTAMP,
+                            1)
+                        .setCell(COLUMN_FAMILY_NAME_STATS, "os_build", TIMESTAMP, "PQ2A.190401.002")
+                        .setCell(COLUMN_FAMILY_NAME_DATA, "data_plan_10gb", TIMESTAMP, "true"))
+                .add(
+                    "phone#5c10102#20190502",
+                    Mutation.create()
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_cell".getBytes()),
+                            TIMESTAMP,
+                            1)
+                        .setCell(
+                            COLUMN_FAMILY_NAME_STATS,
+                            ByteString.copyFrom("connected_wifi".getBytes()),
+                            TIMESTAMP,
+                            0)
+                        .setCell(COLUMN_FAMILY_NAME_STATS, "os_build", TIMESTAMP, "PQ2A.190406.000")
+                        .setCell(COLUMN_FAMILY_NAME_DATA, "data_plan_10gb", TIMESTAMP, "true"));
+
+        dataClient.bulkMutateRows(bulkMutation);
+      }
+    } catch (Exception e) {
```

Comment:
This exception should probably should be thrown - we want it to fail the class setup for tests if something goes wrong.